src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -215,6 +215,12 @@ public void testViewPhotoAndAjaxForLargeScaledSession() throws Exception {
        
         resultsPage.verifyHtmlAjax("/instructorFeedbackResultsAjaxByQuestion.html");
         
+        ______TS("test view photo for view by questions");
+        
+        resultsPage.removeNavBar();
+        resultsPage.hoverClickAndViewPhotoOnTableCell(0, 0, 0, "studentProfilePic?studentemail={*}&courseid={*}&user=CFResultsUiT.instr");
+        resultsPage.hoverClickAndViewPhotoOnTableCell(0, 0, 2, "profile_picture_default.png");
```

Comment:
Hi! I believe I wasnt completely clear on my intensions haha. Combining the functions into one was well done, but keep these as they were (ie hoverClickAndViewGiverPhotoOnTableCell and hoverClickAndViewRecipientPhotoOnTableCell) and in those functions call the new function you have created. That way when someone reads your test cases they understand why you have 2 hover-and-click tests, while at the same time your code is not replicated